namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/resources/athena-iam.tf

```diff
@@ -12,6 +12,24 @@ resource "aws_iam_access_key" "pathfinder-athena-user" {
 }
 
 data "aws_iam_policy_document" "athena" {
+
+  statement {
+    actions = [
+      "athena:ListWorkGroups",
+      "athena:ListEngineVersions",
+      "athena:ListDataCatalogs",
+      "athena:GetCatalogs",
+      "athena:ListDatabases",
+      "athena:GetDatabase",
+      "athena:ListTableMetadata",
+      "athena:GetTableMetadata"
+    ]
+
+    resources = [
+      "*",
```

Comment:
is GetDatabase for '*' a safe setting?

Reply:
@razvan-moj I will try moving that to the statement below and test again. So will probably update this PR.